package.json

```diff
@@ -13,30 +13,27 @@
   "license": "Apache-2.0",
   "author": "Google LLC",
   "bin": {
-    "gapic-generator-typescript": "build/src/start-script.js",
-    "protoc-gen-typescript_gapic": "build/src/cli.js"
+    "gapic-generator-typescript": "build/src/gapic-generator-typescript.js",
+    "protoc-gen-typescript_gapic": "build/src/protoc-plugin.js"
   },
   "files": [
-    "build/src",
-    "pbjs-genfiles",
-    "templates"
+    "build/**/*.js",
```

Comment:
Doesn't this mean you're shipping compiled test output?

Reply:
Hmm, it does. Let me change this.

Reply:
Done.